Gopkg.toml

```diff
@@ -264,3 +264,7 @@
 [[constraint]]
   name = "gopkg.in/DataDog/dd-trace-go.v1"
   version = "1.5.0"
+
+[[constraint]]
+  name = "github.com/shopspring/decimal"
+  branch = "master"
```

Comment:
Please use the released version as 1.1.0 or 1.0.1. Master is updated regularly and so each time it can pick-up latest code from master and so it can break the original functionality.